Makefile.js

```diff
@@ -834,8 +830,8 @@ target.checkRuleFiles = function() {
              */
             const presentHeaders = knownHeaders.filter(header => headers.includes(header));
 
-            for (let i = 0; i < presentHeaders.length; ++i) {
-                if (presentHeaders[i] !== headers[i]) {
+            for (const [i, presentHeader] of Object.entries(presentHeaders)) {
```

Comment:
```suggestion
            for (const [i, presentHeader] of presentHeaders.entries()) {
```

Can we use `Array.prototype.entries` here? It seems unusual to use `Object.entries` on arrays.